src/Package/Repositories/Downloads.php

```diff
@@ -147,33 +130,15 @@ public function create($owner, $repo, $name, $size, $description = '', $content_
 	 * @param   string  $file                   Local file. Example assumes the file existing in the directory
 	 *                                          where you are running the curl command. Yes, the @ matters.
 	 *
-	 * @since  1.0
+	 * @return  boolean
 	 *
-	 * @return boolean
+	 * @note    This API endpoint no longer exists at GitHub
+	 * @since   1.0
+	 * @deprecated  The Releases API should be used instead
 	 */
 	public function upload($key, $acl, $success_action_status, $filename, $awsAccessKeyId, $policy, $signature, $content_type, $file)
 	{
-		// Build the request path.
-		$url = 'https://github.s3.amazonaws.com/';
-
-		$data = array(
-			'key'                   => $key,
-			'acl'                   => $acl,
-			'success_action_status' => (int) $success_action_status,
-			'Filename'              => $filename,
-			'AWSAccessKeyId'        => $awsAccessKeyId,
-			'Policy'                => $policy,
-			'Signature'             => $signature,
-			'Content-Type'          => $content_type,
-			'file'                  => $file
-		);
-
-		// Send the request.
-		$response = $this->client->post($url, $data);
-
-		// @todo Process the response..
-
-		return (201 == $response->code) ? true : false;
+		return true;
```

Comment:
I would definitely throw an exception with a good looking message here.

Returning boolean true to me would indicate that the action I requested has been performed successfully.